alodataset/kitti_stereo_flow_sflow2015.py

```diff
@@ -0,0 +1,385 @@
+
+        # Compute the scene flow from the points cloud
+        mask = mask.any(axis=1)
+        scene_flow = end_points - start_points
+        scene_flow = scene_flow.reshape((size[0], size[1], 3))
+
+        # Not sure it is really usefull but here for security
+        start_occlusion = depth.occlusion.cpu().numpy().astype(bool)
+        end_occlusion = next_depth.occlusion.cpu().numpy().astype(bool)
+
+        # Fusion the occlusion mask
+        mask2 = end_occlusion | start_occlusion
+        mask = mask.reshape((size[0], size[1]))
+        mask2 = mask2.reshape((size[0], size[1]))
+        mask = mask2 | mask
+        mask = Mask(mask, names=("H", "W"))
+
+        return SceneFlow(scene_flow, occlusion=mask)
+
+    def getitem(self, idx):
+        """
+        Function for compatibility
+        Please check at getsequence
+        """
+        if self.sample:
+            return BaseDataset.__getitem__(self, idx)
+        return self.getsequance(idx)
+
+    def getsequance(self, idx) -> Dict[int, Dict[str, Frame]]:
+        """
+        Load a sequence of frames from the dataset.
+
+        Parameters
+        ----------
+        idx : int
+            Index of the sequence.
+
+        Returns
+        -------
+        Dict[int, Dict[str, Frame]]
+            Dictionary of index beetween sequance_start and sequance_end.\n
+            Each index is a dictionary of frames ("left" and maybe "right").
+        """
+        sequence: Dict[int, Dict[str, Frame]] = {}
+        calib = self._load_calib(self.split_folder, idx)
+
+        # We need to load the sequance from the last to the first frame because we need information from the previous
+        # frame to compute the scene flow.
+        for index in range(self.sequence_end, self.sequence_start - 1, -1):
+            sequence[index] = {}
+            sequence[index]["left"] = Frame(os.path.join(self.split_folder, f"image_2/{idx:06d}_{index:02d}.png"))
+            sequence[index]["left"].baseline = calib["baseline"]
+            sequence[index]["left"].append_cam_intrinsic(calib["left_intrinsic"])
+            sequence[index]["left"].append_cam_extrinsic(calib["left_extrinsic"])
+            if "right" in self.load:
+                sequence[index]["right"] = Frame(os.path.join(self.split_folder, f"image_3/{idx:06d}_{index:02d}.png"))
+                sequence[index]["right"].baseline = ["baseline"]
+                sequence[index]["right"].append_cam_intrinsic(calib["right_intrinsic"])
+                sequence[index]["right"].append_cam_extrinsic(calib["right_extrinsic"])
+
+            # Frames at index 10 and 11 are the only one who have ground truth in dataset.
+            if index == 11:
+                if "disp_noc" in self.load:
+                    sequence[11]["left"].append_disparity(
+                        self.load_disp(os.path.join(self.split_folder, f"disp_noc_1/{idx:06d}_10.png")), "disp_noc"
+                    )
```

Comment:
Reading the disparity for image 11 in this way will cause problems, because it is warped to be expressed in the reference frame of image 10. 
This goes against the current convention in alodataset, that the disparity attached to a `Frame` object is expressed in the same coordinate system.

Reply:
@thibo73800 i'm not sure about what should be done here. 
- a user familiar with  KITTI Stereo2015 would expect the disparity of t1 to be warped to t0
- a user familar with aloception would expect `frame_t1.disp` to be the disparity of t1 (not warped)

We could append the disparity to the frame with an explicit name like `"warped_disparity"` to make sure it is not mistakenly used as a normal disparity map.

If we wanted to compute the unwarped disparity, we would need the flow for t1 to t0 (not given) or compute it with a pseudo-inversion (dirty) : none of these solutions seem satisfying.

Reply:
I would suggest to start with warped_disparity for now

Reply:
I named the disparity `warped_disp_noc` and `warped_disp_occ`.